image/spec/specs-go/version_deprecated.go

```diff
@@ -0,0 +1,9 @@
+package v1
+
+import dockerspec "github.com/moby/docker-image-spec/specs-go"
+
+const (
+	Version      = dockerspec.Version      // Deprecated: use [dockerspec.Version].
+	VersionMajor = dockerspec.VersionMajor // Deprecated: use [dockerspec.VersionMajor].
+	VersionMinor = dockerspec.VersionMinor // Deprecated: use [dockerspec.VersionMinor].
+)
```

Comment:
```suggestion
import "github.com/moby/docker-image-spec/specs-go"

const (
	Version      = v1.Version      // Deprecated: use [v1.Version].
	VersionMajor = v1.VersionMajor // Deprecated: use [v1.VersionMajor].
	VersionMinor = v1.VersionMinor // Deprecated: use [v1.VersionMinor].
)
```

Yes, this looks stupid. All other importers of docker-image-spec will also have to deal with it, too. If you don't like it, fix the docker-image-spec module.